src/Assets.php

```diff
@@ -35,7 +35,17 @@ public static function init() {
 	 */
 	public static function register_assets() {
 		$asset_api = Package::container()->get( AssetApi::class );
-		self::register_style( 'wc-block-vendors-style', plugins_url( $asset_api->get_block_asset_build_path( 'vendors-style', 'css' ), __DIR__ ), [] );
+
+		// In the editor, we want our stylesheets to load after editor CSS (see #3068).
+		// This might be no longer necessary when https://github.com/WordPress/gutenberg/issues/20797
+		// is merged.
```

Comment:
Should we create an issue for the backlog for removing this extra code when the fix in Gutenberg lands and we can safely remove because the minimum version of WordPress we support has the fix (if so, you could convert this comment to a todo)?